src/components/_modals/DepositModal/SommelierTab.tsx

```diff
@@ -304,10 +306,14 @@ export const SommelierTab: VFC<DepositModalProps> = (props) => {
                 alignItems="center"
                 href={`https://etherscan.io/tx/${depositResult?.data?.transactionHash}`}
                 isExternal
+                textDecor="underline"
               >
                 <Text as="span">View on Etherscan</Text>
                 <ExternalLinkIcon ml={2} />
               </Link>
+              <Text onClick={doImportToken} textDecor="underline">
```

Comment:
Using `<Text/>` with `onClick` props won't give us clickable behavior. Try adding `as="button"` for better UX.
![CleanShot 2023-01-12 at 00 55 50](https://user-images.githubusercontent.com/39829726/211881989-66caedaa-5a99-49a4-8698-10b8b109b18a.png)

Reply:
Nice Feedback, thanks 🚀